domain/src/main/java/org/oppia/android/domain/topic/TopicListController.kt

```diff
@@ -270,35 +266,15 @@ class TopicListController @Inject constructor(
     return this.addAll(iterable.take(limit - this.getTotalPromotedStoryCount()))
   }
 
-  private fun populateRecentlyPlayedStories(
-    topicProgressList: List<TopicProgress>
-  ): List<PromotedStory> {
-    computePromotedStories(topicProgressList).let {
-      Pair(it.first, it.second)
-      if (it.second < ONE_WEEK_IN_DAYS)
-        return it.first
-    }
-    return emptyList()
-  }
-
-  private fun populateOlderPlayedStories(
-    topicProgressList: List<TopicProgress>
+  private fun computeRecentStoriesList(
```

Comment:
Could we add an in-line comment to clarify that this is for both older recently-played stories as well as more recent recently-played stories? Otherwise maybe `computePlayedStories` might be clearer, something to explain that this is for both recent and older recently-played stories.

Reply:
`computePlayedStories` makes sense. Thanks.

Reply:
Done.